src/test/java/wedlog/address/testutil/GuestBuilder.java

```diff
@@ -139,6 +152,22 @@ public GuestBuilder withoutEmail() {
         return this;
     }
 
+    /**
+     * Sets the {@code RsvpStatus} of the {@code Guest} that we are building.
+     */
+    public GuestBuilder withoutRsvpStatus() {
+        this.rsvpStatus = null;
+        return this;
+    }
+
+    /**
+     * Sets the {@code Dietary Requirements} of the {@code Guest} that we are building.
+     */
+    public GuestBuilder withoutDietaryRequirements() {
+        this.dietaryRequirements = null;
+        return this;
+    }
+
```

Comment:
The guest constructor requires non null values for rsvp status and dietary requirements. Perhaps you could remove these methods?

```suggestion
```